prow/plugins/bugzilla/bugzilla.go

```diff
@@ -438,6 +438,17 @@ func processQuery(query *emailToLoginQuery, email string, log *logrus.Entry) str
 
 func handle(e event, gc githubClient, bc bugzilla.Client, options plugins.BugzillaBranchOptions, log *logrus.Entry) error {
 	comment := e.comment(gc)
+	// check if bug is part of a restricted group
+	if !e.missing {
+		bug, err := getBug(bc, e.bugId, log, comment)
+		if err != nil || bug == nil {
+			return err
+		}
+		if !isBugAllowed(bug, options.AllowedGroups) {
+			// ignore bugs that are in non-allowed groups for this repo
```

Comment:
We don't have to leak which group the bug is in, but we should comment for the user to know that the bug is not in the allowed groups list, and list which groups are allowed. That way they can see feedback and we leak nothing.

Reply:
This would cause us to comment every time there is a change though. Would it make sense to only have it respond about the bug group if it was triggered by a `/bugzilla refresh` and ignore in all other cases?

Reply:
Yeah, that sounds good